src/interface/common.thrift

```diff
@@ -112,5 +112,50 @@ struct Pair {
     2: string value,
 }
 
+/**
+** GOD is A global senior administrator.like root of Linux systems.
+** ADMIN is an administrator for a given Graph Space.
+** DBA is an schema administrator for a given Graph Space.
+** USER is a normal user for a given Graph Space. A User can access (read and write) the data in the Graph Space.
+** GUEST is a read-only role for a given Graph Space. A Guest cannot modify the data in the Graph Space.
+** Refer to header file src/graph/PermissionManager.h for details.
+**/
+
+enum RoleType {
+    GOD    = 0x01,
+    ADMIN  = 0x02,
+    DBA    = 0x03,
+    USER   = 0x04,
+    GUEST  = 0x05,
+} (cpp.enum_strict)
+
+enum UserLoginType {
+   PASSWORD = 0x01,
+   LDAP     = 0x02,
+}
+
+struct RoleItem {
+    1: string        user,
```

Comment:
If we have userID, we'd better use userId here.

Reply:
> If we have userID, we'd better use userId here.

UserID removed.